stream/crowdstrike-endpoint-security/src/requirements.txt

```diff
@@ -1,3 +1,3 @@
 pycti==6.4.2
-prometheus-client==0.21.1
+prometheus-client~=0.20.0
```

Comment:
fix dependency resolution issue (renovate bot update to 0.21 bu pycti only compatible with 0.20)